gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/connection.js

```diff
@@ -165,6 +169,21 @@ class Connection extends EventEmitter {
         };
       }
 
+      if (this._session === true && op !== 'authentication') {
+        if (bytecode !== null) {
+          reject(new Error('do not support bytecode in session mode'));
+          return;
+        }
+        if (args === null) {
+          reject(new Error('get nil args in request'));
+          return;
+        }
+
+        const message = Buffer.from(this._header + JSON.stringify(this._getSessionRequest(requestId, null, args)))
```

Comment:
Missing `;` at the end of the statement.

Reply:
sorry for this obvious mistake